src/js/actions/auth-actions.js

```diff
@@ -0,0 +1,15 @@
+export const SET_AUTH = 'SET_AUTH';
+export const RESET_AUTH = 'RESET_AUTH';
+
+export function setAuth(props) {
+  return {
+    type: SET_AUTH,
+    props: props
+  };
+}
+
+export function resetAuth() {
```

Comment:
Ok, thanks. Wasn't sure where it was used, but then realized it was to initialize the smooch core module.